src/protocol/http/org/apache/jmeter/protocol/http/control/CacheManager.java

```diff
@@ -152,22 +197,18 @@ private boolean hasVaryHeader(URLConnection conn) {
      *            result to decide if result is cacheable
      */
     public void saveDetails(HttpResponse method, HTTPSampleResult res) {
```

Comment:
This is _old_ code and a result of the different implementations of the http samplers. What refactoring do you have in mind?